src/extensions/ElementalAreaTextSearchExtension.php

```diff
@@ -24,7 +29,7 @@ public function getAllElements()
      * as well as virtual elements
      * to extract the complete list of element presents within a single area
      *
-     * @return DataList
+     * @return array<int|string>
```

Comment:
Return type here was wrong, this function returns an array of ids so `array<int|string>`